src/coreclr/jit/emitarm64.cpp

```diff
@@ -31254,6 +31495,36 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insLatency    = PERFSCORE_LATENCY_2C;
             break;
 
+        case IF_SVE_GG_3A: // ........ii.mmmmm ......nnnnnddddd -- SVE2 lookup table with 2-bit indices and 16-bit
```

Comment:
Nit: Should we squish these all together into one case block?

Reply:
So for this one, because these instructions need fixing, it is possible that the values will be different for each instruction, so I don't necessarily want to collapse them yet. I did something similar for other formats already,